openedge-plugin/src/main/java/org/sonar/plugins/openedge/sensor/OpenEdgeProparseSensor.java

```diff
@@ -246,6 +249,26 @@ private void parseMainFile(SensorContext context, InputFile file, RefactorSessio
         LOG.error("Unable to parse " + file.relativePath() + " - IOException was caught - Please report this issue", caught);
       }
       return;
+    } catch (RecognitionException caught) {
+      LOG.error("Error during code parsing for " + file.relativePath() + " at position " + caught.getFilename() + ":"
+          + caught.getLine() + ":" + caught.getColumn(), caught);
+      numFailures++;
+      NewIssue issue = context.newIssue().forRule(
+          RuleKey.of(Constants.STD_REPOSITORY_KEY, OpenEdgeRulesDefinition.PROPARSE_ERROR_RULEKEY));
+      NewIssueLocation loc = issue.newLocation().on(file).message(Strings.nullToEmpty(caught.getMessage()) + " in "
+          + caught.getFilename() + ":" + caught.getLine() + ":" + caught.getColumn());
+      if (file.relativePath().equals(caught.getFilename())) {
+        try {
+          TextPointer strt = file.newPointer(caught.getLine(), caught.getColumn() - 1);
+          TextPointer end = file.newPointer(caught.getLine(), caught.getColumn());
+          loc.at(file.newRange(strt, end));
+        } catch (IllegalArgumentException uncaught) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1166)